packages/notifi-react/lib/context/NotifiHistoryContext.tsx

```diff
@@ -35,6 +35,8 @@ export type NotifiHistoryContextType = {
   historyItems: HistoryItem[];
   unreadCount: number;
   hasNextPage: boolean;
+  isIncludeRead: boolean;
```

Comment:
As mention above, do you think it is a good idea to make this `isIncludeRead` as internal const instead of expose it out?